services/ui-src/src/utils/errors/errorHandler.tsx

```diff
@@ -1,6 +1,12 @@
-export const errorHandler = (error: Error | any): void => {
-  const isError = error && error.stack && error.message;
-  const message = isError ? error.message : JSON.stringify(error);
-  alert(message);
-  return message;
+export const errorHandler = (
+  error: Error | any,
+  callback?: Function,
+  message?: string
+): void => {
+  const errorData = {
+    name: error.name,
+    message: message || error.message,
+  };
+  if (callback) callback(errorData);
+  console.log(errorData.name, errorData.message); // eslint-disable-line no-console
 };
```

Comment:
Rewrote the `errorHandler` 'cause it is being used very differently now. No need for the `isError` check anymore and we're handling a callback now, etc.